src/ensembl/src/content/app/about-ensembl/AboutEnsembl.tsx

```diff
@@ -7,11 +7,33 @@ import styles from './AboutEnsembl.scss';
 const AboutEnsembl = () => {
   return (
     <div className={styles.aboutEnsembl}>
-      <DocLayout mainContent={<Page />} sidebarContent={<Links />} />
+      <DocLayout
+        appHeaderContent={<AppHeaderContent />}
+        searchBoxContent={<SearchBoxContent />}
+        globalNavContent={<GlobalLinks />}
+        mainContent={<Page />}
+        localNavContent={<LocalLinks />}
+      />
     </div>
   );
 };
 
+const AppHeaderContent = () => {
+  return (
+    <>
```

Comment:
Here and below — the purpose of React fragment (written as `<> </>`) is to enable returning from a React component an array of elements without wrapping them in a single parent.

E.g.: previously

```
<div>
  <Element1 />
  <Element2 />
</div>
```

with fragments (fragments do not get rendered into DOM; so using them reduces the number of not strictly necessary DOM elements):

```
<>
  <Element1 />
  <Element2 />
</>
```

But in the PR fragments do not play this role; so they are not necessary.

Reply:
I wasn't sure what to do with the placeholder content, as the code was refusing to compile with multiple HTML elements until I put in the fragment tags.